index.html

```diff
@@ -1,71 +1,121 @@
+                <a title="contribute :)" href="https://github.com/JermaSites/JermaDebt" class="github-corner"
+                    aria-label="View source on GitHub"><svg width="12.2vh" height="12.2vh" viewBox="0 0 250 250"
+                        style="fill:#151513; color:#fff; position: absolute; top: 0; border: 0; right: 0;"
+                        aria-hidden="true">
+                        <path d="M0,0 L115,115 L130,115 L142,142 L250,250 L250,0 Z"></path>
+                        <path
+                            d="M128.3,109.0 C113.8,99.7 119.0,89.6 119.0,89.6 C122.0,82.7 120.5,78.6 120.5,78.6 C119.2,72.0 123.4,76.3 123.4,76.3 C127.3,80.9 125.5,87.3 125.5,87.3 C122.9,97.6 130.6,101.9 134.4,103.2"
+                            fill="currentColor" style="transform-origin: 130px 106px;" class="octo-arm"></path>
+                        <path
+                            d="M115.0,115.0 C114.9,115.1 118.7,116.5 119.8,115.4 L133.7,101.6 C136.9,99.2 139.9,98.4 142.2,98.6 C133.8,88.0 127.5,74.4 143.8,58.0 C148.5,53.4 154.0,51.2 159.7,51.0 C160.3,49.4 163.2,43.6 171.4,40.1 C171.4,40.1 176.1,42.5 178.8,56.2 C183.1,58.6 187.2,61.8 190.9,65.4 C194.5,69.0 197.7,73.2 200.1,77.6 C213.8,80.2 216.3,84.9 216.3,84.9 C212.7,93.1 206.9,96.0 205.4,96.6 C205.1,102.4 203.0,107.8 198.3,112.5 C181.9,128.9 168.3,122.5 157.7,114.1 C157.9,116.9 156.7,120.9 152.7,124.9 L141.0,136.5 C139.8,137.7 141.6,141.9 141.8,141.8 Z"
+                            fill="currentColor" class="octo-body"></path>
+                    </svg></a>
+            </div>
+            <a class="head-link" href="https://debt.jerma.io">Jerma Debt Tracker</a>
+        </h1>
+    </header>
+    <div class="data-display">
+        <h1 id="debtCounterNum">$</h1>
+        <h2 id="debtCounterText">
+            Fifty Four Billion,<br>
+            Five Hundred and Forty-Nine Million,<br>
+            Three Hundred and One Thousand,<br>
+            Four Hundred and Thirty<br>
+        </h2>
+    </div>
+    <article>
+        <div class="notice">
+            <div class="container">
+                <a href="https://www.strawpoll.me/42356670">Poll: How long until invalid entries are removed from
+                    debt.jerma.io?</a>
+            </div>
+        </div>
+        <div class="intro">
+            <div class="container">
+                On the 8th of November, at approximately 6pm, popular e-clown Jerma985 bet his chat, while playing the
+                game
+                Noita, that if he missed the next shot on the the next enemy, he would owe chat 500 million community
+                subs.
+                <br>
+                Jerma985 then proceeded miss the next shot he took. This brings his debt to chat to an original two and
+                a
+                half billion US dollars.<br>
+                <br>This website will track as Jerma slowly whittles away his debt to chat
+                <br>This guy is fucked.
```

Comment:
Bad formatting. stuff that is not broken should not be a new line. it should be 
```
<br> blahblah.....
<br> this is the next actual broken line of text
```

Reply:
There is a max line, which is industry standard.